pkg/config/config.go

```diff
@@ -499,6 +502,10 @@ func initConfig(config Config) {
 	config.SetKnown("apm_config.stats_writer.queue.exp_backoff_growth_base")
 	config.SetKnown("apm_config.analyzed_rate_by_service.*")
 	config.SetKnown("apm_config.analyzed_spans.*")
+	config.SetKnown("apm_config.log_throttling")
+	config.SetKnown("apm_config.bucket_size_seconds")
```

Comment:
This one is never used and can be considered internal. It’s fine to remove it.

Reply:
As long as it can be used (eg: it does something in the code) I think it's better to not treat it as "unknown".

Reply:
I don’t agree. We will remove this. It is never used and we shouldn’t advertise it. But if you want to keep it, I don’t mind.

Reply:
By putting it here we don't advertise it more than by having it in the apm config.go itself. Let's remove it from both places when it can be removed (for 6.13?).

Reply:
No problem 👍